### ISSUE TITLE
pls 

### ISSUE THREAD

**Issue by SyncMaster93** [rom1v/sndcpy#101]
my sndcpy doesnt work
![image](https://user-images.githubusercontent.com/26763758/103941466-2f12b500-512f-11eb-874e-b3feb0ea1549.png)
help

**Comment by rom1v** [comment]
https://github.com/rom1v/sndcpy/issues/39#issuecomment-732794953

**Comment by SyncMaster93** [comment]
nope.. doesnt work.. is there any way to reset all of that.. or should i reset my pc

**Comment by rom1v** [comment]
Why do you mean "does not work"? What did you do exactly? What is the result?

**Comment by SyncMaster93** [comment]
> Why do you mean "does not work"? What did you do exactly? What is the result?

deleting my sndcpy.apk from phone and it closing my console
